codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt

```diff
@@ -312,12 +303,8 @@ class BuilderProcessor(private val codeGenerator: CodeGenerator, val logger: KSP
             // Add a function that composes a component with variants given a node name that
             // contains all the variant properties and values
             out.appendText("    @Composable\n")
-            // Add an override declaration if this interface inherits from another
-            val overrideDecl = if (overrideInterface.isNotEmpty()) "override " else ""
-            out.appendText("    ${overrideDecl}fun CustomComponent(\n")
-            // Default parameters are not allowed when overriding a function
-            val defaultModifierDecl = if (overrideInterface.isEmpty()) " = Modifier" else ""
-            out.appendText("        modifier: Modifier$defaultModifierDecl,\n")
+            out.appendText("    final fun CustomComponent(\n")
+            out.appendText("        modifier: Modifier,\n")
```

Comment:
Seems like you lose the ` = Modifier` bit here.

Reply:
Yea I don't think that's needed. This function is called from generated code and a modifier is always passed in.